boards/arm/stm32f0l0g0/stm32g071b-disco/include/board.h

```diff
@@ -171,7 +171,7 @@
  *   SPI1_SCK  - PA1
  */
 
-#undef  GPIO_SPI1_MISO                  /* Not used */
+#define GPIO_SPI1_MISO 0                /* Not used - simplex tx */
```

Comment:
Should "Not used" be removed?

Reply:
MISO is not used in simplex tx mode, but this is now handled by SPI driver:
https://github.com/apache/incubator-nuttx/blob/3c0a3dabfce4c5da866ee25966c3125bba8a0453/arch/arm/src/stm32f0l0g0/stm32_spi.c#L1944-L1947
It must be defined to prevent compiler's error.

Reply:
Thanks for explanation